### ISSUE TITLE
Inquiry Commands

### ISSUE THREAD

**Issue by Tomas1337** [misterhay/VISCA-IP-Controller#1]
Thanks for your repo! Been experimenting with PTZ cameras.

No commands  yet for Inquiring Camera parameters such as Position / Camera Focus position.

**Comment by misterhay** [comment]
Yes, that's definately something we should get working. I [experimented a little](https://github.com/misterhay/VISCA-IP-Controller/blob/main/old/listen_to_camera.py) with it at the beginning, but it hasn't been a priority.

Of course pull requests are welcome :)

**Comment by Tomas1337** [comment]
Yup! Will pull once I make some progress. But even the first step of getting proper commands from my PTZ Camera is worrisome.

The only responses I get from any command are incrementing `01110003000000019051ff`. 
Can't figure out why it wont give me a proper response even if its a inquiry packet I sent.

**Comment by cathedralstreamer** [comment]
Hello @Tomas1337 and @misterhay , I have made some progress with the same idea (at least for my cameras). See  #32
